waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -107,7 +106,12 @@ npmDepsForWasp spec =
             ("react-dom", "^17.0.2"),
             ("@tanstack/react-query", "^4.13.0"),
             ("react-router-dom", "^5.3.3"),
-            ("react-scripts", "5.0.1")
+            ("react-scripts", "5.0.1"),
+            -- todo make optional
```

Comment:
TODO
and maybe explain what to make optional -> all the TS stuff

Reply:
Will do.